heal/vlmd/extract/conversion.py

```diff
@@ -36,20 +39,25 @@ def _detect_input_type(filepath, ext_to_input_type=ext_map):
 
 def convert_to_vlmd(
     input_filepath,
-    input_type=None,
-    data_dictionary_props=None,
+    input_type: str = None,
+    data_dictionary_props: dict = None,
+    include_all_fields: bool = True,
```

Comment:
we don't need this to be a CLI parameter right?

Reply:
Correct, leaving the default as True, the SDK will mimic the current behavior of `healdata-utils` for filling in all fields for CSV conversions. We could later add it to the CLI if there were requests from users.